2024-05-06 21:42:36,766 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-05-06 21:42:36,766 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Configure stats pid to 3108
2024-05-06 21:42:36,767 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-05-06 21:42:36,767 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Loading settings from /root/mats/sae/wandb/settings
2024-05-06 21:42:36,767 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-06 21:42:36,767 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-06 21:42:36,767 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'sae/main.py', 'program_abspath': '/root/mats/sae/main.py', 'program': '/root/mats/sae/main.py'}
2024-05-06 21:42:36,767 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-06 21:42:36,767 INFO    MainThread:3108 [wandb_init.py:_log_setup():521] Logging user logs to /root/mats/sae/wandb/run-20240506_214236-24t3m0xg/logs/debug.log
2024-05-06 21:42:36,767 INFO    MainThread:3108 [wandb_init.py:_log_setup():522] Logging internal logs to /root/mats/sae/wandb/run-20240506_214236-24t3m0xg/logs/debug-internal.log
2024-05-06 21:42:36,767 INFO    MainThread:3108 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f74877f7f40>
2024-05-06 21:42:36,768 INFO    MainThread:3108 [wandb_init.py:init():561] calling init triggers
2024-05-06 21:42:36,768 INFO    MainThread:3108 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-05-06 21:42:36,768 INFO    MainThread:3108 [wandb_init.py:init():611] starting backend
2024-05-06 21:42:36,768 INFO    MainThread:3108 [wandb_init.py:init():615] setting up manager
2024-05-06 21:42:36,769 INFO    MainThread:3108 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-05-06 21:42:36,770 INFO    MainThread:3108 [wandb_init.py:init():623] backend started and connected
2024-05-06 21:42:36,776 INFO    MainThread:3108 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-05-06 21:42:36,777 INFO    MainThread:3108 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-06 21:42:36,777 INFO    MainThread:3108 [wandb_init.py:init():715] updated telemetry
2024-05-06 21:42:36,785 INFO    MainThread:3108 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-05-06 21:42:37,073 INFO    MainThread:3108 [wandb_run.py:_on_init():2357] communicating current version
2024-05-06 21:42:37,220 INFO    MainThread:3108 [wandb_run.py:_on_init():2366] got version response 
2024-05-06 21:42:37,220 INFO    MainThread:3108 [wandb_init.py:init():799] starting run threads in backend
2024-05-06 21:42:37,517 INFO    MainThread:3108 [wandb_run.py:_console_start():2335] atexit reg
2024-05-06 21:42:37,517 INFO    MainThread:3108 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-05-06 21:42:37,517 INFO    MainThread:3108 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-05-06 21:42:37,517 INFO    MainThread:3108 [wandb_run.py:_redirect():2280] Redirects installed.
2024-05-06 21:42:37,518 INFO    MainThread:3108 [wandb_init.py:init():842] run started, returning control to user process
2024-05-06 21:42:38,986 INFO    MainThread:3108 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-06 21:42:38,987 INFO    MainThread:3108 [wandb_init.py:_pause_backend():432] pausing backend
2024-05-06 21:42:55,693 INFO    MainThread:3108 [wandb_init.py:_resume_backend():437] resuming backend
2024-05-06 21:42:55,702 INFO    MainThread:3108 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-06 21:42:55,702 INFO    MainThread:3108 [wandb_init.py:_pause_backend():432] pausing backend
2024-05-06 21:42:57,421 INFO    MainThread:3108 [wandb_init.py:_resume_backend():437] resuming backend
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Configure stats pid to 3108
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Loading settings from /root/mats/sae/wandb/settings
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'sae/main.py', 'program_abspath': '/root/mats/sae/main.py', 'program': '/root/mats/sae/main.py'}
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_init.py:_log_setup():521] Logging user logs to /root/mats/sae/wandb/run-20240506_214257-96vlz71b/logs/debug.log
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_init.py:_log_setup():522] Logging internal logs to /root/mats/sae/wandb/run-20240506_214257-96vlz71b/logs/debug-internal.log
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_init.py:init():561] calling init triggers
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-05-06 21:42:57,437 INFO    MainThread:3108 [wandb_init.py:init():586] re-initializing run, found existing run on stack: 24t3m0xg
2024-05-06 21:42:57,440 INFO    MainThread:3108 [wandb_run.py:_finish():2064] finishing run jacobcd52/sae/24t3m0xg
2024-05-06 21:42:57,440 INFO    MainThread:3108 [jupyter.py:save_history():445] not saving jupyter history
2024-05-06 21:42:57,440 INFO    MainThread:3108 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-06 21:42:57,440 INFO    MainThread:3108 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-05-06 21:42:57,440 INFO    MainThread:3108 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-05-06 21:42:57,440 INFO    MainThread:3108 [wandb_run.py:_restore():2287] restore
2024-05-06 21:42:57,440 INFO    MainThread:3108 [wandb_run.py:_restore():2293] restore done
2024-05-06 21:43:00,407 INFO    MainThread:3108 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-05-06 21:43:00,407 INFO    MainThread:3108 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-05-06 21:43:00,420 INFO    MainThread:3108 [wandb_run.py:_footer_sync_info():3895] logging synced files
